negar/gui.py

```diff
@@ -1,10 +1,17 @@
-#!/usr/bin/python
+#!/usr/bin/env python3
```

Comment:
This discovery depends on the system installation. Nowadays most of the modern OSs are dropping version 2 family and `python` refers to version 3.

Reply:
fixed.